src/browser/api/external_application.ts

```diff
@@ -10,7 +10,27 @@ import { Identity } from '../../shapes';
 import * as ProcessTracker from '../process_tracker.js';
 import route from '../../common/route';
 
-const authenticatedConnections: any[] = [];
+const authenticatedConnections: ExternalApplicationOptions[] = [];
+
+export interface ExternalApplicationClient {
+    type: string;
+    version: any;
+}
+
+export interface ExternalApplicationOptions extends Identity {
+    runtimeClient: boolean;
```

Comment:
Is a top-level boolean the best way to classify this?
Consider instead `ExternalApplicationOptions.client.type` which should contain this information as well.